<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEditSaida">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>441</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>659</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonAbrirArquivo">
       <property name="text">
        <string>Abrir Arquivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCriarNovoArquivo">
       <property name="text">
        <string>Criar Novo Arquivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonIncluirNoFinal">
       <property name="text">
        <string>Incluir no Final do Arquivo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLimparCaixa">
       <property name="text">
        <string>Limpar Caixa de Texto</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox_Ordem">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>OrdemCrescente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OrdemDecrescente</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_Ordenar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_SaidaGrid">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>149</number>
    </attribute>
    <column>
     <property name="text">
      <string>Matr√≠cula</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nome</string>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="comboBox_NomesOUNumeros">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Nomes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Matriculas</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>658</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
